src/components/subscriptions/SubscriptionDetails.jsx

```diff
@@ -15,23 +15,26 @@ export default function SubscriptionDetails() {
             <small>Purchase Date</small>
           </div>
           <div className="lead">
-            {moment(details.purchaseDate).format('MMMM D, YYYY')}
+            {moment(details.purchase_date)
+              .format('MMMM D, YYYY')}
```

Comment:
nit: Might be nice to make this string format a constant as it seems to be shared for all our dates

Reply:
👍  found a helper method for this [here](https://github.com/edx/frontend-app-admin-portal/blob/master/src/utils.js#L14-L19)

Reply:
Nice! Forgot that helper function existed haha